File: python/명예의 전당1.py
def solution(k, score):
    stack = []
    answer = []
    for i in range(len(score)):
        stack.append(score[i])
        stack.sort(reverse=True)
        if len(stack) == k+1:
            stack.pop()
        answer.append(stack[-1])
    return answer

File: js/명예의 전당1.js
function solution(k, score) {
    var stack = [];
    var answer = [];
    for (let i=0; i<score.length; i++) {
        stack.push(score[i]);
        stack.sort((a,b) => b-a);
        if (stack.length === k+1) {
            console.log(stack.pop());
            stack.pop();
        }
        answer.push(stack[stack.length-1]);
    }
    return answer;
}